edge/pkg/metamanager/client/pod.go

```diff
@@ -44,7 +44,7 @@ func (c *pods) Update(cm *corev1.Pod) error {
 }
 
 func (c *pods) Delete(name, options string) error {
-	resource := fmt.Sprintf("%s/%s/%s", c.namespace, model.ResourceTypePod, name)
+	resource := fmt.Sprintf("%s/%s/%s", c.name, model.ResourceTypePod, name)
```

Comment:
What's the meaning of the c.name here? If pods from different namespaces with same name, does this will lead to conflict?

Reply:
I think it is a dead code, i do not find where to use namespace(maybe i miss something), we can just keep this part as an empty string.